tests/src/Functional/EventIntegrationTest.php

```diff
@@ -132,12 +133,15 @@ public function testCreateEvent(): void {
     $assert_session->responseContains('text-0.txt');
     $assert_session->pageTextContains('Sat, 01/22/2022 - 02:12');
     $assert_session->pageTextContains('Thu, 02/24/2022 - 20:00');
+    $assert_session->pageTextContains('https://europa.eu');
+    $assert_session->elementExists('css', '.field--name-oe-sc-event-registration-url > .field__item > a[href="https://europa.eu"]');
```

Comment:
Don't these two lines test the same thing now?
Or is the link text of registration url also 'https://europa.eu`?

Reply:
So yeah actually this is the link text.
We could make this more explicit to actually compare the link text instead of just using pageTextContains.
But this would be a waste of time. Going to approve :)